### PR TITLE
Generic Mining Missions

### PR DESCRIPTION
**Pull request by rovermicrover**
## Blocked By/Requires
https://github.com/endless-sky/endless-sky/pull/7761

## Summary
Adds generic mining missions to human space. They require you to go and get X amount of a mineral and then return the planet with it.

## Build
You can find a build of this PR with an added generic mining mission that uses outfitter [here](https://github.com/rovermicrover/endless-sky/releases/tag/mining-missions).

### CODE REVIEW

**Comment by rovermicrover** [comment]
I would like for these missions to be in Human space as a way to encourage early game interest in mining. Though maybe save the largest size missions for other civilizations?

File: data/dialog phrases.txt [hunk 1]
```diff
@@ -131,3 +131,28 @@ phrase "generic aborted pirate cargo mission"
...
+	word
+		`Your client on <destination>`
+	word
+		` `
+		` gratefully `
+	word
+		`pays you <payment> for helping to`
+	word
+		` delivering `
+		` procuring `
```
Comment (bene-dictator) [inline data/dialog phrases.txt:145]:
```suggestion
		` deliver `
		` procure `
```
A little bit of grammar.

File: data/dialog phrases.txt [hunk 2]
```diff
@@ -131,3 +131,28 @@ phrase "generic aborted pirate cargo mission"
...
+		`the requested `
+	word
+		`<outfit-units> of <outfit>.`
+
+phrase "generic mineral visit dialog"
+	word
+		`You arrived at <destination>, `
+	word
+		`but you don't have the `
+		`you have yet to mine the `
```
Comment (bene-dictator) [inline data/dialog phrases.txt:156]:
```suggestion
		`but you don't have `
		`you have yet to mine `
```

**Comment by quyykk** [comment]
This PRs commits are all over the place lmao. I think it's probably better to wait until the code PR is merged before opening content that depends on it? Or at least not including the other PR in this one and marking this one a draft?

**Comment by rovermicrover** [comment]
The PR opens up with a Blocked By Statement. I can convert to draft if you want to make it clearer that it should not be merged.

The commits are all from the other branch. I am not squash merging between branches to try and avoid merge issues.

**Comment by rovermicrover** [comment]
This branch is all borked now closing out and will re open later

**Pull request closed without merging**